.vscode/tasks.json

```diff
@@ -336,6 +336,22 @@
 			"dependsOn": [
 				"K3D - Install prerequisites"
 			]
+		},
+		{
+			"label": "Get example app from SDK package",
+			"detail": "Get example app from SDK package.",
+			"type": "shell",
+			"command": "rm -rf app/ && cp -a ~/.local/lib/python3.9/site-packages/sdv_examples/${input:exampleApp} app && pip install -r ./app/requirements.txt",
```

Comment:
Move this to a shell script and check for git changes. Show error message if the the git working directory is not clean in order to avoid overwriting the exited changes.
--> Message: Please commit or stash your changes before importing the example app otherwise all changes will be discarded.